bin/cli-flags.js

```diff
@@ -37,6 +37,14 @@ module.exports = {
       describe: 'Do not refresh page if HMR fails',
       group: ADVANCED_GROUP,
     },
+    {
+      name: 'setup-exit-signals',
+      type: Boolean,
+      describe: 'Close and exit the process on SIGINT and SIGTERM',
+      group: ADVANCED_GROUP,
+      defaultValue: true,
+      negative: true,
+    },
     {
       name: 'stdin',
       type: Boolean,
```

Comment:
Another issue is `stdin`. Should it be handled by `webpack-cli` or added to `options` and handled in `dev-server` side like `setupExitSignals`?

Reply:
@ylemkimon sorry for some delay, to be honest, I don't quite remember how this option works, ideally stdin can be not only for webpack serve

Reply:
It just calls `process.exit()` on `Ctrl+D` (EOF): https://github.com/webpack/webpack-dev-server/blob/4ab1f21bc85cc1695255c739160ad00dc14375f1/lib/utils/createConfig.js#L83-L90

Now I think it should not just exit the process, but close the server, it should be on the dev-server's side, like `setupExitSignals`.

Reply:
hm, yes, also I think it should be supports not only for webpack serve, ref:
- https://github.com/webpack/webpack-cli/issues/789
- https://github.com/webpack/webpack-cli/issues/885

I think for v4 webpack-cli `--stdin` is broken, so I think we should fix it in both places :disappointed:

Reply:
What do you think? I am working on https://github.com/webpack/webpack-cli/issues/2121, then will review https://github.com/webpack/webpack-cli/pull/2126 and return to this, maybe we csn start to solve problem with `--stdin` parallels

Reply:
I've added `stdin` option along with `setupExitSignals`.